xml/art_sle_ha_install_quick.xml

```diff
@@ -788,6 +793,98 @@ softdog                16384  1</screen>
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-ha-inst-quick-test-with-cluster-script">
+    <title>Testing with the &test.script; Script</title>
+    <para>
+     The command &cmd.test.script; is a tool to run standardized tests
+     for a cluster.
+     It triggers cluster failures and verify configuration to find
```

Comment:
verify -> verifies the cluster configuration

Reply:
Fixed.